design-documents/graph-ql/coverage/custom-attributes/attributes-metadata.graphqls

```diff
@@ -0,0 +1,222 @@
+type Query {
+    customAttributeMetadataV2(attributes: [AttributeMetadataInput!]!): CustomAttributeMetadata
```

Comment:
A question here if we should require at least one attribute or no attributes will return all schema